ios/src/main/java/org/ergoplatform/ios/api/IosEncryptionManager.kt

```diff
@@ -0,0 +1,47 @@
+package org.ergoplatform.ios.api
+
+import org.ergoplatform.api.AesEncryptionManager
+import org.ergoplatform.utils.LogUtils
+import org.robovm.apple.localauthentication.LAContext
+import kotlin.random.Random
+
+object IosEncryptionManager {
+    private const val KEYCHAIN_SERVICE = "org.ergoplatform.ios"
+    private const val KEYCHAIN_ACCOUNT = "ergoWalletKey"
+    private const val KEY_LENGTH = 28
```

Comment:
Please leave a comment in the code on why it is 28.

Reply:
Done